docker-entrypoint.njs

```diff
@@ -2,6 +2,24 @@
 
 // override the config.json with ENV variables
 var config = require('./config.json');
+var configIstex = require('./config-istex.json');
+var configCnrs = require('./config-cnrs.json');
+
+  
+
+ 
+
```

Comment:
Don't you use eslint?
Our eslint rules scream when there are too much blank lines.